pcdsdevices/mpod_apalis.py

```diff
@@ -246,6 +292,50 @@ def set_current_ramp_speed(self, ramp_speed):
         """
         self.current_ramp_speed.put(ramp_speed)
 
+    @property
+    def polarity(self) -> Polarity:
+        model_str = self.model.get()
+        if model_str.endswith("x"):
+            return Polarity.BIPOLAR
+        elif model_str.endswith("p"):
+            return Polarity.POSITIVE
+        elif model_str.endswith("n"):
+            return Polarity.NEGATIVE
+
+        raise ValueError(f"Unable to determine module polarity: {model_str}")
+
+    @property
+    def limit_scales(self) -> tuple[float, float]:
+        """
+        Returns limit scaling tuple.  When multiplied by the channel's max_voltage
+        signal (:VoltageNominal) will produce the actual limits for the channel
+
+        Returns
+        -------
+        tuple[float, float]
+            The voltage limits as a proportion of the max_voltage
+        """
+        pos_limit = self.limit_pos.get()
+        neg_limit = self.limit_neg.get()  # positive % here means negative limit
```

Comment:
Similar to above, if we cache these (and raise if missing) we can't hit the same sort of queue-blocking issues that the LODCM has.